modules/tinymce/src/plugins/table/main/ts/ui/Helpers.ts

```diff
@@ -72,16 +73,16 @@ export type CellData = {
   readonly backgroundcolor?: string;
 };
 
-const rgbToHex = (dom: DOMUtils) => (value: string): string =>
-  Strings.startsWith(value, 'rgb') ? dom.toHex(value) : value;
+const rgbToHex = (value: string): string =>
+  Strings.startsWith(value, 'rgb') ? Transformations.rgbaToHexString(value) : value;
 
 const extractAdvancedStyles = (dom: DOMUtils, elm: Node): AdvancedStyles => {
   const element = SugarElement.fromDom(elm);
   return {
     borderwidth: Css.getRaw(element, 'border-width').getOr(''),
     borderstyle: Css.getRaw(element, 'border-style').getOr(''),
-    bordercolor: Css.getRaw(element, 'border-color').map(rgbToHex(dom)).getOr(''),
-    backgroundcolor: Css.getRaw(element, 'background-color').map(rgbToHex(dom)).getOr('')
+    bordercolor: Css.getRaw(element, 'border-color').map(rgbToHex).getOr(''),
```

Comment:
Actually, this was a mistake - `rgbToHex` does need the `dom` argument for the moment. It could in theory be untangled, but I think we leave that for later tech debt.

Reply:
Actually... that's incorrect as dom is completely unused here as `rgbToHex` doesn't use it at all now. I've pushed the fix up after talking to Matt about it.

Reply:
I don't know what code I was looking at on Friday then. I can only assume it was out of date. The code had a compile error because `rgbToHex` still required a `dom` argument.